influxql/INFLUXQL.md

```diff
@@ -229,6 +229,7 @@ ALTER RETENTION POLICY policy1 ON somedb DURATION 1h REPLICATION 4
 
 ```
 create_continuous_query_stmt = "CREATE CONTINUOUS QUERY" query_name on_clause
+                               [ "RESAMPLE" [ "EVERY" duration_lit ] [ "FOR" duration_lit ] ]
```

Comment:
Need to fix this.  It allows `CREATE CONTINUOUS QUERY foo ON bar RESAMPLE BEGIN...`